### PR TITLE
Heretic blades now do 20 damage instead of 17

### PR DESCRIPTION
**Pull request by ANAME593**
## About The Pull Request
Heretic blades will now do 3+ damage making it a more viable melee option

## Why It's Good For The Game
because i feel as if 17 is too low for a antag melee, by changing it to 20 it will now take 5 hits to down unarmoured targets instead of 6.  blades are still quite useful however heretic is a very difficult antag especially after the rework so i think this change is fair

## Changelog

:cl:
balance: heretic blades do 20 damage up from 17, have 35% armor penetration, but deal 20% less damage to structures like lockers
/:cl:

### CODE REVIEW

**Comment by moocowswag** [comment]
I think it might still take 6 hits if the target is at perfect mood (thinks are never truely consistent in this game)

**Comment by Shadowflame909** [comment]
When the loud valid (he antag) sword comes out. You sure hope it downs your opponent before they can scream their heads off and summon a valid hunter army at your location.

Especially if your target in itself is an armored sec officer, or a gamer gear'd head of staff. (Which it will be as a heretic!)

**Review by MrMelbert — CHANGES_REQUESTED** [review]
20 force puts it up a threshold and allows them to break open lockers, which gives me pause

Suggestion: Maybe 18 or 19 force and armor penetration (50%)? Or give it a negative demolition modifier (0.8)?

Otherwise I'm 100% okay with a damage buff for blades, I had one planned but didn't push it yet

**Comment by Shadowflame909** [comment]
please do the armor penetration one since the majority of heretic targets are armored

**Comment by ANAME593** [comment]
> Suggestion: Maybe 18 or 19 force and armor penetration (50%)? Or give it a negative demolition modifier (0.8)?
18-19 force still makes it take around  6 hits to down a unarmoured target so i can just do the demolition modifier thing which ill try figure out how to do

**Comment by moocowswag** [comment]
Armor penetration is nice but 50% IS ALOT and armor is already in a sad enough state as it is.

But while we are here I want to bring in some statistics to make it easier to choose between the two options presented.


My knowledge of how armor and armor pen works is probably entirely false so I`m gonna say right here ASSUMING THAT 50% armor pen = armor is halved

At 26.09% melee armor both the 50% armor pen 17 damage blade and the 20 damage blade will deal 14.783 damage (I think)

So I`m pretty sure single digits of melee armor dont come unless its multiples of 5 though so pretty much at 30% melee armor or more the 50% armor pen blade is stronger.

Sec jumpsuit gives 10% melee armor, vest gives 35 for a total of 45 to torso and helmet gives 35 for a total of 35 (jumpsuit doesnt cover head) but you can definitely hit the arms and legs of an average seccie pretty hard for 18 damage with the proposed blade.

Since we are here, for fun, I`ll include a specific build that many heretics might face (android chaplain) which has like 50 melee armor and -5 brute damage
The 20 damage blade will deal 5 brute damage
The 17 damage 50% armor pen blade will deal 7.5

If chaplain has the audacity to wear a security jumpsuit under his retextured riot armor for a total of 60 melee armor and -5 brute damage modifier.
The 20 damage blade will deal 3 brute
The 17 damage 50% armor pen blade will deal 6.9 (nIce)

(Keep in mind all of this math is probably horribly wrong because I do not know for exactly sure how armor works)

**Comment by ANAME593** [comment]
> Armor penetration is nice but 50% IS ALOT and armor is already in a sad enough state as it is due to baton sticks (But oh boy is is armor pen necessary against specific foes, it would be nice if we had sweeping **_nerfs to armor in exchange for letting them mitigate batons_**)
This pr is about heretic blades but there is a thing called "energy armour" which basically makes batons respect armour because you could have level 10 protection on everything but still lose to a baton

**Comment by Shadowflame909** [comment]
no wonder antag blades as strong as crud. an antagonist doing 3 brute to a target is hillariously awful

**Comment by ANAME593** [comment]
i dont want blades doing 17 damage, so im gonna add armour pen to the 20 damage blade therefore it can somewhat deal with armoured & unarmoured targets instead of like a tradeoff where we can choose 17 damage + 50% armour pen or 20 damage + 0% armour pen

**Comment by Shadowflame909** [comment]
20 damage + armor pen but with the downside of no locker/crate/door breaking like other 20 damage weapons

pretty good weapon

**Comment by moocowswag** [comment]
> > This pr is about heretic blades but there is a thing called "energy armour" which basically makes batons respect armour because you could have level 10 protection on everything but still lose to a baton

I thought energy armor only mitigates disablers??? And regardless if we are bringing up armor pen we are bringing up armor and we are bringing up the forbidden meta weapons that overshadow most weapons in the game, including the one we are buffing.

**Comment by ANAME593** [comment]
> > > This pr is about heretic blades but there is a thing called "energy armour" which basically makes batons respect armour because you could have level 10 protection on everything but still lose to a baton
> 
> I thought energy armor only mitigates disablers??? 

on other codebases there is a thing where baton respects energy armour

**Comment by Fikou** [comment]
50 armor penetration penetrates everything besides high tier mining gear

**Comment by ANAME593** [comment]
> 50 armor penetration penetrates everything besides high tier mining gear

What about 35? thats the same as eswords, and we want it to have good armour pen

File: code/modules/antagonists/heretic/items/heretic_blades.dm [hunk 1]
```diff
@@ -12,12 +12,15 @@
...
-	force = 17
+	force = 20
 	throwforce = 10
+	demolition_mod = 0.8
 	hitsound = 'sound/weapons/bladeslice.ogg'
+	armour_penetration = 35
 	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "rends")
 	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "rend")
 	var/after_use_message = ""
+
```
Comment (CursedBirb) [inline code/modules/antagonists/heretic/items/heretic_blades.dm:23]: That one more empty line is not needed

**Comment by moocowswag** [comment]
> > 50 armor penetration penetrates everything besides high tier mining gear
> 
> What about 35? thats the same as eswords, and we want it to have good armour pen

36 is desword 30 is esword

35 is darkened cult longsword.

**Comment by moocowswag** [comment]
Oh so is armor pen a flat decrease to armor and not a %?

**Review by MrMelbert — APPROVED** [review]

**Pull request merged**